src/app/portlets/content-types/edit/content-types-edit.component.ts

```diff
@@ -154,7 +154,7 @@ export class ContentTypesEditComponent implements OnInit, OnDestroy {
      * @memberof ContentTypesEditComponent
      */
     setTemplateInfo(): void {
-        this.dotMessageService.messageMap$.subscribe(() => {
+        this.dotMessageService.messageMap$.pipe(takeUntil(this.destroy$)).subscribe(() => {
```

Comment:
Why this is not a `take(1)`? we only need this one time right? is not like the messages are going to update over time or am I missing something?

Reply:
?